src/main/java/org/jabref/gui/DragAndDropHelper.java

```diff
@@ -8,7 +8,7 @@
 
 import javafx.scene.input.Dragboard;
 
-import org.jabref.logic.util.io.FileUtil;
+import org.jabref.logic.util.FileUtil;
```

Comment:
Why move it a level up? In the io package, there are many other libraries - and I like the grouping

![image](https://user-images.githubusercontent.com/1366654/229381450-0b6ab18b-d833-4c48-b264-220143edf026.png)